src/coord/src/coord/dataflow_builder.rs

```diff
@@ -180,13 +180,13 @@ impl<'a> DataflowBuilder<'a> {
                         // to the collection to arrange it ourselves.
                         let indexes = &self.catalog.indexes()[on_id];
                         if let Some((id, _)) = indexes.iter().find(|(_id, keys)| keys == key_set) {
-                            dataflow.add_index_import(*id, index_desc, typ.clone(), *view_id);
+                            dataflow.import_index(*id, index_desc, typ.clone(), *view_id);
                         }
                     }
                 }
             }
         });
-        dataflow.add_view_to_build(*view_id, view.clone(), view.typ());
+        dataflow.add_view_to_build(*view_id, view.clone());
```

Comment:
would you like to change `add_view_to_build` to `build_view`?

Reply:
I wasn't sure about that. I'd love to find a verb, but `build` sounds very imperative and like "ok you are a builder, so build this". Maybe `include_view` or something?

Reply:
`generate_view`?

Or `connect_view`/`link_view`? To reflect the fact the views are intermediate steps to get from the import to the export?